### ISSUE TITLE
Ability to create multiple VMs in parallel

### ISSUE THREAD

**Issue by jay7x** [jay7x/puppet-lima#10]
While starting multiple VMs in parallel is unsafe in general (see https://github.com/lima-vm/lima/issues/1354) it'd be really nice to have a way to 🤞🏻, 🙏🏻 and do it anyway. As long as all VMs are created already it should be safe to start them in parallel. So my idea is to add a `parallel=true` kind of parameter to the `lima::cluster::start` plan.

**Comment by jay7x** [comment]
Just realized that start process is already parallelized.. So two possible things to implement:

1. Add `threads=N` parameter to specify how many VMs to start in parallel (fallback to half of CPU count as it is now)
2. Add `parallel=true` which will create VMs in parallel too (if you're sure that all files were downloaded before)
